portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -141,3 +160,13 @@ async function fetchSubmittedLocations() {
 
   return commentData;
 }
+
+function allowUserSubmit() {
+  fetch('/allow-user-submit')
```

Comment:
Like Kenneth said I think this should be done in the backEnd to prevent users from modifying it.